src/locales/en.js

```diff
@@ -16,17 +16,21 @@ const en = {
       size: 'How big is your team?',
       officeType: 'What type of office are you looking for?',
       name: 'What is the name of your company?',
-      peers: 'Would you like to share office with others from your industry?'
+      peers: 'Would you like to share office with others from your industry?',
+      location: 'Where would you like to be situated?',
     },
     answer: {
-      email: 'My email is: ',
-      size: 'We are a group of: ',
-      officeType: 'We need: ',
+      email: 'My email is... ',
+      size: 'We are a group of... ',
+      officeType: 'We need an... ',
```

Comment:
are you shure about three dots ? If you don't like: maby we should get rid of both

Reply:
Removed